### ISSUE TITLE
Data in application/x-www-form-urlencoded format from github isn't handled

### ISSUE THREAD

**Issue by weltling** [FriendsOfPHP/pickleweb#38]
Currently only JSON data is accepted from github, supporting application/x-www-form-urlencoded  might be subject of the further refactoring/extending.

**Comment by pierrejoye** [comment]
Hm?

In which context? Hook? Auth?

We may add the right header. Also the builtin server is bad at handling
json data or other mime types.

**Comment by weltling** [comment]
Still talking about the hooks, as after figuring out the cause of #37 i was just trying to misconfigure as much as i could. But this could be done abstracting the data parsing into some layer according to their type. So I wanted just to note it could/should be done somewhen :)

**Comment by pierrejoye** [comment]
Does it work once configured correctly?

This is the main thing, every thing else can wait.

We can a short sanity check in the github controller and bailout with a
message saying that this event is not supported. But that's all we need for
now.

**Comment by weltling** [comment]
Yep, when `application/json` selected and it works. When documented, it's should be fine at the current stage.

**Comment by pierrejoye** [comment]
Woohoo :)

**Comment by pierrejoye** [comment]
@anatol

Can you be more precised in the wiki? Create/delete tag, and release

Also I have to fix the API key and link part, it is easier now, one click
and you get it from the extension page.

**Comment by weltling** [comment]
But that's exactly what stands there on the hook settings page, against the checkbox:

Create
Branch, or tag created. 

Exactly one word against checkbox, so hard to mix up.

Branch is bailed out by pickleweb, just tested and it works.

**Comment by pierrejoye** [comment]
Yes, but the word "tag" is missing in the wiki :)
